src/content/resources/cpp/multidimensional-arrays.md

```diff
@@ -0,0 +1,63 @@
+---
+authors:
+  - "alearori#8661"
+created_at: 2019/11/15
+title: "Multidimensional Arrays"
+---
+
+## What is a multidimensional array?
+
+To put simply, a multidimensional array is an array that contains more than one row and column. It is commonly used for making matricies, but it can also be used for storing information such as if you use `string`. The format of the container is `{{a1,a2,a3,...},{b1,b2,b3,...},{c1,c2,c3,...},...}`, where `a1`,`a2`,`a3`, and etc. are objects of the data type that was array was declared to contain at the start.
+
+### It is NOT an array inside of an array
+
+It just looks like one. Unlike with other container types, an array cannot hold anything other than the original data type, so if you declare it to be of data type `int` at the start, it cannot hold anything else except integers inside. If you're looking to use multiple data types, you should make a `class`, `struct` or use other container types like `map` that fulfill your requirement.
+
+### How do we make one?
+
+To create multidimensional array, you do it like a regular array but with a extra `[]` at the end. 
+
+#### Method 1: 
+Use `data_type array_name[A][B]` where `data_type` is an data type(such as `int`, `char` or etc.), `array_name` is what the name of the array will be and `A` and `B` are the integers.
```

Comment:
"Are the integers" doesn't add up if we've just mentioned data_type can be anything like int/char.

Reply:
A and B are the things inside the `[]` during declaration, but I see there was also a mistake in the grammar that could confuse people, so I corrected that and expanded on it more.